bundles/ch.elexis.core/src/ch/elexis/core/utils/CoreUtil.java

```diff
@@ -40,20 +40,20 @@ public static enum OS {
 	 * </ul>
 	 * Requires boolean parameter.
 	 */
-	public static final String TEST_MODE = "elexis.test.mode";
-
+	public static final String TEST_MODE = "elexis.test.mode"; //$NON-NLS-1$
+	public static final String HOME_MODE = "ch.elexis.home"; //$NON-NLS-1$
```

Comment:
Das ist nun doppelt definiert in `CoreUtil` und in `CoreHub`.  Bitte nur an 1ner Stelle definieren.